examples/units/ellipse_with_units.py

```diff
@@ -30,7 +30,7 @@
     ])
 
 
-x, y = np.dot(R, np.array([x, y]))
+x, y = np.dot(R, [x, y])
```

Comment:
```suggestion
x, y = R @ [x, y]
```
Simpler?

Reply:
I'd rather leave the dot -> @ for another time.